lib/trino-hive-formats/src/test/java/io/trino/hive/formats/FormatTestUtils.java

```diff
@@ -632,4 +666,9 @@ public static LineBuffer createLineBuffer(Slice value)
         lineBuffer.write(value.getInput(), value.length());
         return lineBuffer;
     }
+
+    public static boolean isScalarType(Type type)
```

Comment:
You can use `!HiveUtil.isStructuralType(type)`

Reply:
That class is in the Hive plugin which depends on this module.